sw.js

```diff
@@ -1,103 +1,47 @@
 );
 
-// Make this always match package.json version
-const version = '0.1.3';
-const cacheName = `defaultCache_${version}`;
-
-const { cacheableResponse, precaching, routing, runtimeCaching } = goog;
-
-const router = new routing.Router();
-const localhost = registration.scope;
-const staticCache = new precaching.UnrevisionedCacheManager({ cacheName });
-const requestWrapper = new runtimeCaching.RequestWrapper({ cacheName });
-const cdnRequestWrapper = new runtimeCaching.RequestWrapper({
-  plugins: [
-    new cacheableResponse.CacheableResponsePlugin({ statuses: [0, 200] })
-  ],
-  cacheName
-});
-
-/**
- * Route for local CSS and JS assets
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#stale-while-revalidate
- */
-const assetRoute = new routing.RegExpRoute({
-  regExp: new RegExp(`^${localhost}.*\\.(css|js|png|svg)$`),
-  handler: new runtimeCaching.StaleWhileRevalidate({ requestWrapper })
-});
+// Cache fallback for HTML files
+workbox.routing.registerRoute(
+  /(?:^[^.]+|\.html)$/,
+  workbox.strategies.networkFirst()
+);
 
-/**
- * Route for assets on cloudfront.net CDN
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#stale-while-revalidate
- */
-const cdnAssetRoute = new routing.RegExpRoute({
-  regExp: new RegExp('^https://.*\\.cloudfront\\.net/.*\\.(png|svg)$'),
-  handler: new runtimeCaching.StaleWhileRevalidate({
-    requestWrapper: cdnRequestWrapper
+// Cache CSS and JS files
+workbox.routing.registerRoute(
+  /\.(?:js|css)$/,
+  workbox.strategies.staleWhileRevalidate({
+    cacheName: 'static-resources'
   })
-});
-
-/**
- * Route for pages
- *
- * Strategy:
- * https://developers.google.com/web/fundamentals/instant-and-offline/
- * offline-cookbook/#network-falling-back-to-cache
- */
-const navRoute = new routing.NavigationRoute({
-  whitelist: [/./],
-  handler: new runtimeCaching.NetworkFirst({ requestWrapper })
-});
-
-/**
- * Precache resources
- */
-staticCache.addToCacheList({
-  unrevisionedFiles: ['/', '/404/', '/error/']
-});
-
-self.oninstall = event => {
-  event.waitUntil(Promise.all([staticCache.install().then(skipWaiting)]));
-};
-
-self.onactivate = event => {
-  caches.keys().then(keys => {
-    keys.forEach(cacheName => {
-      if (!cacheName.endsWith(version)) {
-        caches.delete(cacheName);
-      }
-    });
-  });
-  return event.waitUntil(clients.claim());
-};
-
-router.registerRoutes({
-  routes: [assetRoute, cdnAssetRoute, navRoute]
-});
+);
 
-/**
- * Fallback route for stuff that doesn't match
- */
-router.setDefaultHandler({
-  handler: new runtimeCaching.NetworkOnly()
-});
+// Cache images
+workbox.routing.registerRoute(
+  /\.(?:png|gif|jpg|jpeg|svg)$/,
+  workbox.strategies.cacheFirst({
+    cacheName: 'images',
+    plugins: [
+      new workbox.expiration.Plugin({
+        maxEntries: 60,
+        maxAgeSeconds: 30 * 24 * 60 * 60 // 30 Days
```

Comment:
Curious how you landed at a max entry amount of 60 and a max age of 30 days? It feels like a guessing game without something to lean on but was wondering if you had referenced a best practice you found. :)

Reply:
Wondering if @grigs has any thoughts on this for the images cache? :)

Reply:
I used the example from https://developers.google.com/web/tools/workbox/guides/common-recipes. I think that 30 days is reasonable, but maybe we could remove the `maxEntries`?

Reply:
Sounds good. Keeping what you have works for me. 😉